static/src/javascripts/projects/commercial/modules/third-party-tags/krux.spec.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import { url, shouldRun } from 'commercial/modules/third-party-tags/krux';
+import config from 'lib/config';
+
+describe('Krux', () => {
+    beforeEach(() => {
+        config.switches = {
+            krux: true,
+        };
+    });
+
+    it('should not load if switch is off', () => {
+        config.switches.krux = false;
+
+        expect(shouldRun).toBeFalsy();
```

Comment:
How about strict true/false rather than truthy/falsy? Otherwise shouldRun could be undefined and still pass

`expect(shouldRun).toBe(false);`